Source/WebCore/platform/mediastream/gstreamer/GStreamerMediaStreamSource.cpp

```diff
@@ -1079,22 +1063,25 @@ void webkitMediaStreamSrcAddTrack(WebKitMediaStreamSrc* self, MediaStreamTrackPr
     auto* element = source->get();
     gst_bin_add(GST_BIN_CAST(self), element);
 
-    auto pad = adoptGRef(gst_element_get_static_pad(element, "src"));
-    auto tags = mediaStreamTrackPrivateGetTags(*track);
-    if (!onlyTrack) {
```

Comment:
Ah, forgot to remove this now un-needed onlyTrack argument. I'll update the patch.